service-provider.md

```diff
@@ -3,4 +3,116 @@ description: Service Bindings for Service Providers
 permalink: /service-provider/
 ---
 
-# TODO
+Service providers expose bindings through a Secret resource with data required for connectivity.  The specification's [Provisioned Service section][provisioned-service] describes this in detail.  But there are two important parts:
+
+1. A Secret resource with data required for connectivity
+2. A custom resource with `.status.binding.name` attribute pointing to the Secret resource
```

Comment:
I think we should specify that (2) is optional, and considered best practice if a CR is involved, as you could directly point to a Secret from a `ServiceBinding` CR.

Reply:
I have updated the text based on this feedback.